v2/internal/pkg/operator/filtered_collector.go

```diff
@@ -325,7 +325,7 @@ func (o FilterCollector) filterOperator(ctx context.Context, op v2alpha1.Operato
 		return v2alpha1.CatalogFilterResult{
 			OperatorFilter:     op,
 			FilteredConfigPath: filterConfigDir,
-			ToRebuild:          false,
+			ToRebuild:          true,
```

Comment:
Is this field ill-named? Why would we rebuild the catalog if it's already been filtered previously?

Reply:
@r4f4 - yep crappy naming convention IMHO